dev/TabView/TabView.cpp

```diff
@@ -635,18 +737,36 @@ void TabView::OnItemsChanged(winrt::IInspectable const& item)
                 m_updateTabWidthOnPointerLeave = true;
                 if (args.Index() == TabItems().Size())
                 {
-                    UpdateTabWidths(true,false);
+                    UpdateTabWidths(true, false);
                 }
             }
         }
         else
         {
+            if (args.CollectionChange() == winrt::CollectionChange::ItemInserted && listViewInnerSelectedIndex != -1)
+            {
+                // UpdateBottomStrokes repeats in both if statements
+                // I can't wrap my head around on how to write it better
+                // Thoughts?
```

Comment:
You could store the difference between indexes in a variable and query for the absolute value between them, then you can nest an if to verify whether the difference is positive or not, that way you can save one call to UpdateBottonStrokes
```
auto difference = itemIndex - selectedIndex;

if (abs(difference) <= 1)
{
    if (difference > 0)
    {
        SetTabViewItemAdjacentState(selectedIndex + 2, L"NotAdjacent");
    }
    else
    {
        SetTabViewItemAdjacentState(selectedIndex - 2, L"NotAdjacent");
    }

    UpdateBottomStrokes(SelectedIndex());
}
```